Packs/PAN-OS/Integrations/Panorama/README.md

```diff
@@ -48,7 +48,11 @@ This integration was integrated and tested with versions 8.xx, 9.xx, and 10.xx o
    * [pan-os-query-logs](#pan-os-query-logs)
    * [pan-os-check-logs-status](#pan-os-check-logs-status)
    * [pan-os-get-logs](#pan-os-get-logs)
+   * [pan-os-get-master-key-details](#pan-os-get-master-key-details)
+   * [pan-os-create-master-key](#pan-os-create-master-key)
+   * [pan-os-update-master-key](#pan-os-update-master-key)
 * The target argument is supported only in operational type commands. Meaning, you cannot use it with commit, logs, or PCAP commands.
+* After running the `pan-os-create-master-key` or `pan-os-update-master-key` command, the current API key will become invalid and a new one needs to be obtained. Refer to the [documentation on how to get an API key](https://docs.paloaltonetworks.com/pan-os/11-0/pan-os-panorama-api/get-started-with-the-pan-os-xml-api/get-your-api-key).
 
```

Comment:
```suggestion

Creating or updating the encryption master key of Palo Alto Networks Firewall or Panorama invalidates the current API key and requires obtaining a new one. All subsequent commands will raise an "Invalid Credentials" error until a new API key is obtained and the integration instance is updated accordingly.

```

Reply:
Thank you for your feedback. The sentence has been updated based on your suggestion in commit 85dab2f2c6d1f0308f5d274059664b920a3ce75a.